### PR TITLE
Eth nic gro

### PR DESCRIPTION
**Pull request by lessan98**

### CODE REVIEW

**Review by dgalan-xxia — CHANGES_REQUESTED** [review]
- let's add (update existing test) that covers all params
- move eth nic settings under a profile
- update python package version

File: model/dataflow/host_management.yaml [hunk 1]
```diff
@@ -10,78 +10,11 @@ components:
...
-          type: integer
-          description: RX buffer size in bytes
-        nic_tx_buffer:
-          type: integer
-          description: TX buffer size in bytes
-        nic_speed:
-          type: integer
-          description: configured link speed Mbps (e.g. 100000 for 100G)
-        management_address:
+        eth_nic_profile:
```
Comment (dgalan-xxia) [inline model/dataflow/host_management.yaml:13]: eth_nic_profile_name

File: model/dataflow/host_management.yaml [hunk 2]
```diff
@@ -10,78 +10,11 @@ components:
...
+          description: |-
+            The pod profile associated with the pod(s).
```
Comment (dgalan-xxia) [inline model/dataflow/host_management.yaml:15]: update description

File: model/dataflow/dataflow.yaml [hunk 3]
```diff
@@ -19,6 +19,92 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Dataflow.FlowProfile"
+        eth_nic_profiles:
```
Comment (dgalan-xxia) [inline model/dataflow/dataflow.yaml:22]: I think this should go under host management

File: model/profiles/eth_nic_settings.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Profiles.EthNicSetting:
```
Comment (dgalan-xxia) [inline model/profiles/eth_nic_settings.yaml:3]: looks good, just one observation, we need to make a better hierarchy, like profiles.dataflow.host_management.ethnicsettings

Comment (lessan98) [reply model/profiles/eth_nic_settings.yaml:3]: isn t the point of profiles to keep them in a general area and allow their usage anywhere else? i know that right now each profile has only a specific region where it should be, but i m thinking that it s better to leave this flexibility in the future, to be able to add any profile anywhere if needed.

**Pull request merged**